GithubEmojiCompletions.py

```diff
@@ -19,14 +21,40 @@ def on_query_completions(self, view, prefix, locations):
         pt = locations[0] - len(prefix) - 1
         ch = view.substr(sublime.Region(pt, pt + 1))
 
+        complete_with_emoji = view.settings().get('github_emoji_complete_with_emoji') is True
+
         # emoji completions
         if ch == ':':
-            return settings.get("emojiCompletions")
+            if complete_with_emoji:
+                return emojis_emoji
+            else:
+                return emojis_alias
         # emoji completions for commit messages
         if ch == '@':
-            return settings.get("commitEmojiCompletions")
+            if complete_with_emoji:
+                return commit_emojis_emoji
+            else:
+                return commit_emojis_alias
+
         return []
 
+    def on_post_text_command(self, view, cmd, args):
+        if cmd != 'commit_completion' \
+            or view.settings().get('github_emoji_complete_with_emoji') is not True:
+            return
+
+        # the second condition is used to check whether it's a buggy emoji which
+        # is printed as more than one character
+
+        actual_emojis = list(map(lambda emo: emo[1], emojis))
+        for region in view.sel():
```

Comment:
Is this for emojis like this one `[":arrow_backward: ◀️\tblack left-pointing triangle", "arrow_backward:"]`?